sdk/js/src/CacheStore.ts

```diff
@@ -24,17 +31,64 @@ export abstract class CacheStore {
         return this.store?.getItem(storeKey)
     }
 
+    private getConfigStoreKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonymousConfig : StoreKey.IdentifiedConfig
+    }
+
+    private getUserIdStoreKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonUserId : StoreKey.IdentifiedConfigUserId
+    }
+
+    private getConfigFetchDateKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonymousConfigDate : StoreKey.IdentifiedConfigDate
+    }
+
+    private saveConfigUserId(user: DVCPopulatedUser): void {
+        const { user_id: userId, isAnonymous } = user
+        const userIdKey = this.getUserIdStoreKey(user)
+        this.save(userIdKey, userId)
+        this.logger?.info(`Successfully saved ${isAnonymous ? 'anonymous': 'identified'} user id to local storage`)
+    }
+
+    private saveConfigFetchDate(user: DVCPopulatedUser): void {
+        const fetchDateKey = this.getConfigFetchDateKey(user)
+        this.save(fetchDateKey, now())
+        this.logger?.info('Successfully saved config fetch date to local storage')
+    }
```

Comment:
we don't need to expose these, the client should never be calling these methods directly

Reply:
yeah you can do everything in saveConfig. Also we don't have to log that we saved the config user id and fetch date, logging just the config being saved is enough